mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala

```diff
@@ -24,20 +24,23 @@ import org.apache.spark.annotation.Since
 import org.apache.spark.ml.Model
 import org.apache.spark.ml.attribute.NominalAttribute
 import org.apache.spark.ml.param._
-import org.apache.spark.ml.param.shared.{HasHandleInvalid, HasInputCol, HasOutputCol}
+import org.apache.spark.ml.param.shared.{HasHandleInvalid, HasInputCol, HasInputCols, HasOutputCol}
 import org.apache.spark.ml.util._
 import org.apache.spark.sql._
 import org.apache.spark.sql.expressions.UserDefinedFunction
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types.{DoubleType, StructField, StructType}
 
 /**
- * `Bucketizer` maps a column of continuous features to a column of feature buckets.
+ * `Bucketizer` maps a column of continuous features to a column of feature buckets. Since 2.3.0,
+ * `Bucketizer` can also map multiple columns at once. Whether it goes to map a column or multiple
+ * columns, it depends on which parameter of `inputCol` and `inputCols` is set. When both are set,
+ * a log warning will be printed and by default it chooses `inputCol`.
```

Comment:
We should probably also mention that `splits` is only used for single column and `splitsArray` for multi column